tbb.sh

```diff
@@ -23,4 +23,9 @@ MODULEDIR="$INSTALLROOT/etc/modulefiles"
 MODULEFILE="$MODULEDIR/$PKGNAME"
 mkdir -p "$MODULEDIR"
 alibuild-generate-module --bin --lib > $MODULEFILE
+cat >> "$MODULEFILE" <<EOF
+# extra environment
+set TBB_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
```

Comment:
Do you need a `TBB_ROOT` variable separately or can you use `PKG_ROOT` provided by `alibuild-generate-module`?

Reply:
probably not needed but it's anyway not exported. I copied the pattern from other recipes.

Reply:
Fair enough. I'll merge tonight when the Linux CI is done, but before the nightly tag starts.

Reply:
Well, we should probably fix all of them. Anyways, since this is broken let's go ahead with it. @TimoWilken make sure the cache is hot as well as this will rebuild ROOT.